docs/content/preview/deploy/multi-dc/async-replication.md

```diff
@@ -184,255 +182,266 @@ When both universes use different certificates, you need to store the certificat
+3. Execute the following command to set up the replication stream using the bootstrap IDs generated in step 1. Ensure that the bootstrap IDs are in the same order as their corresponding table IDs.
 
-  {{< note title="Note" >}}
-  It is important that the bootstrap_ids are in the same order as their corresponding table_ids!
-  {{< /note >}}
+      ```sh
+    ./bin/yb-admin -master_addresses <target_universe_master_addresses> setup_universe_replication \
+      <source_universe_uuid>_<replication_stream_name> <source_universe_master_addresses> \
+      <comma_separated_source_universe_table_ids> <comma_separated_bootstrap_ids>
+      ```
 
-  ```sh
-  ./bin/yb-admin -master_addresses <target_universe_master_addresses> setup_universe_replication \
-  <source_universe_uuid>_<replication_stream_name> <source_universe_master_addresses> \
-  <comma_separated_source_universe_table_ids> <comma_separated_bootstrap_ids>
-  ```
+    Consider the following example:
 
-  For Example:
-  ```sh
-  ./bin/yb-admin-master_addresses 127.0.0.11:7100,127.0.0.12:7100,127.0.0.13:7100 setup_universe_replication \
-  00000000-1111-2222-3333-444444444444_xCluster1 127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 \
-  000033e1000030008000000000004000,000033e1000030008000000000004003,000033e1000030008000000000004006 \
-  fb156717174941008e54fa958e613c10,a2a46f5cbf8446a3a5099b5ceeaac28b,c967967523eb4e03bcc201bb464e0679
-  ```
+      ```sh
+    ./bin/yb-admin-master_addresses 127.0.0.11:7100,127.0.0.12:7100,127.0.0.13:7100 setup_universe_replication \
+      00000000-1111-2222-3333-444444444444_xCluster1 127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 \
+      000033e1000030008000000000004000,000033e1000030008000000000004003,000033e1000030008000000000004006 \
+      fb156717174941008e54fa958e613c10,a2a46f5cbf8446a3a5099b5ceeaac28b,c967967523eb4e03bcc201bb464e0679
+      ```
+
+You can modify the bootstrap as follows:
+
+- To wipe the test setup, use the `delete_universe_replication` command.
+- After running the `bootstrap_cdc_producer` command, you can verify entries via the `list_cdc_streams` command.
+
+You can also perform the following modifications:
+
+- To add a table to the source and target universes, use the `alter_universe_replication add_table` command.
+- To remove an existing table from the source and target universes, use the `alter_universe_replication remove_table` command.
+- To change the master nodes on the source universe, execute the `alter_universe_replication set_master_addresses` command.
+- You can verify changes via the `get_universe_config` command.
 
-## Schema migration
+## Migrate schema
 
-This section describs how to execute DDL operations, after replication has been already configured for some tables.
+You can execute DDL operations, after replication has been already configured for some tables.
```

Comment:
You can execute DDL operations, even after replication has been already configured for some tables.
or
You can execute DDL operations after replication has been already configured for some tables.
?